_test/test_change_crystal/test_change_crystal_1.m

```diff
@@ -9,9 +9,8 @@
 
 % -----------------------------------------------------------------------------
 % Add common functions folder to path, and get location of common data
-horace_root = horace_root();

```

Comment:
why delete the local cache of `horace_root()`
Since `<horace_root>/_test` is a common base to both these paths that could be created once and `common_xxx` added.